src/botPage/bot/Interface/ToolsInterface.js

```diff
@@ -1,13 +1,38 @@
 import CandleInterface from './CandleInterface';
 import MiscInterface from './MiscInterface';
 import IndicatorsInterface from './IndicatorsInterface';
+import { translate } from '../../../common/i18n';
 
 // prettier-ignore
 export default Interface => class extends IndicatorsInterface(
     MiscInterface(CandleInterface(Interface))) {
     getToolsInterface() {
         return {
-            getTime: () => parseInt(new Date().getTime() / 1000),
+            getTime   : () => parseInt(new Date().getTime() / 1000),
+            toDateTime: (timestamp) => {
+                let dateTime;
+                if(typeof timestamp === 'number') {
+                    dateTime = new Date(timestamp * 1000);
+                    return `${[
+                        dateTime.getFullYear().toString(),
```

Comment:
You don't have to explicitly call the `toString()` function here as the numeric value returned by `dateTime.getFullYear()` will be converted to a string automatically.